src/Components/IFrame/index.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import {
+  container,
+  frame
+} from './style.module.css';
+
+const IFrame = ({
+  url,
+  title = 'iframeContent',
+  frameBorder = 0,
+}) => {
+  return (
+    <div className={container}>
+      <iframe
```

Comment:
Does this need to set width and height attrs for the iframe? Usually you have to hack those a bit to make it scale responsively

Reply:
This was taken from the responsive vimeo component lauren had made and i was trusting that, but does need testing.